hal/halx86/apic/tsccal.S

```diff
@@ -69,20 +69,20 @@ FUNC TscCalibrationISR
     rdtsc
 
     /* Read the current phase */
-    movzx rcx, byte ptr [TscCalibrationPhase]
+    movzx rcx, byte ptr [rip+TscCalibrationPhase]
```

Comment:
Yes. Otherwise it emits a R_X86_64_32S relocation, and this breaks everything.
See here: https://stackoverflow.com/questions/6093547/what-do-r-x86-64-32s-and-r-x86-64-64-relocation-mean